packages/x-components/build/docgen/templates/docgen-component.template.js

```diff
@@ -0,0 +1,24 @@
+function createComponentTemplate(renderedUsage, doc) {
+  const { displayName, description, docsBlocks } = doc;
+  const title = kebabToPascalCase(displayName);
+  return `
+  ---
+  title: ${title}
+  ---
+  # ${title}
+
+  ${description || ''}
```

Comment:
So I understand that you are overwritting the default template for the component and with this line we avoid the `>` quotation formatting

Reply:
I think we are also fixing the third point of [this task](https://searchbroker.atlassian.net/browse/EX-5063)

Reply:
Accidentally, but yes, we should be solving those issues.